cmd/cored/main.go

```diff
@@ -50,6 +51,7 @@ import (
 const (
 	httpReadTimeout  = 2 * time.Minute
 	httpWriteTimeout = time.Hour
+	latestVersion    = "1.0.1"
```

Comment:
Per conversation: `1.0.1` is accurate here. This value represents the version we have already reached, not the one we are looking forward to. It will tick to `1.0.2` once we're satisfied that the release is ready.